testing/docker/cloudbuild.yaml

```diff
@@ -14,6 +14,10 @@
 
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: [ 'build', '--build-arg', 'BASE_IMAGE=golang:1.19', '-t', 'gcr.io/$PROJECT_ID/go119', '.' ]
+    env:
```

Comment:
Just curious, is this needed?

Reply:
nope! thanks for noticing. i'll remove it.